app/utils/fetch-live-paginated-collection.js

```diff
@@ -0,0 +1,31 @@
+import LivePaginatedCollection from 'travis/utils/live-paginated-collection';
+
+// Fetches a LivePaginatedCollection instance
+//
+// It will use store.filter function in order to get a base for pagination and
+// then it will create a paginated collection with a content set to the filtered
+// collection.
+let fetchLivePaginatedCollection = function (store, modelName, queryParams, options) {
+  let dependencies = options.dependencies || [],
+    filter = filter || (() => true),
```

Comment:
Did you mean this to be `filter = options.filter || (() => true),`?

Reply:
Yup, I fixed it [later](https://github.com/travis-ci/travis-web/commit/8e155969afd78ce2362c72c7c3ca50ad65e72f28). Tests didn't catch it, because I tested the filtering capabilities only in `store.filter` tests, but not on `store.paginated` tests.

Reply:
Oops, I didn’t see that followup PR, thanks 😄